Gemfile

```diff
@@ -33,10 +34,13 @@ gem "ransack"
 gem "sass-rails", ">= 6"
 gem "simplecov"
 gem "simplecov-rcov"
+gem "simple_token_authentication"
 gem "sqlite3", "~> 1.4"
 gem "turbolinks", "~> 5"
 gem "webpacker", "~> 5.0"
 gem "whenever"
+gem "jwt"
+gem "rack-cors"
```

Comment:
2 gem này làm gì e?

Reply:
Dạ... + gem "jwt" dùng để mã hóa chuỗi json thành 1 đoạn mã để đảm bảo tính an toàn.
+ gem "rack-cors" dùng để hỗ trợ chia sẻ tài nguyên chéo (CORS) cho các ứng dụng web tương thích với Rack.